mbed.h

```diff
@@ -69,7 +69,7 @@
 #include "drivers/I2CSlave.h"
 #include "drivers/Ethernet.h"
 #include "drivers/CAN.h"
-#include "drivers/RawSerial.h"
+#include "drivers/UnbufferedSerial.h"
```

Comment:
Actually removing `RawSerial` from here is probably a bit too breaking?

I assume `UARTSerial` is becoming `BufferedSerial` in a follow-up PR?

Reply:
Ok, will add `UnbufferedSerial` and leave `RawSerial`.

> I assume UARTSerial is becoming BufferedSerial in a follow-up PR?

Not quite, `BufferedSerial` will be added and `UARTSerial` will be marked deprecated.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/01dac5b91aa1541aaf03a45701d43130fe3d5d90..6f17d09bc079d4788ce1c087c7236a96d66f4dec).